### PR TITLE
Separate proto into multiple files

### PR DESCRIPTION
**Pull request by shifucun**
The number of protobuf and API has increased dramatically in the past. This PR try to separate based on logical function. Also

* Remove unused protobuf
* Remove old GetPlaceStatVarsUnion API which is not used any more.

### CODE REVIEW

File: proto/landing.proto [hunk 1]
```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Google LLC
```
Comment (beets) [inline proto/landing.proto:1]: naming nit: landing_page.proto to match the route?

File: proto/misc.proto [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  int32 max_results = 2;
+}
+
+// Search response from mixer.
+message SearchResponse {
+  // Matching entities broken down by type.
+  repeated SearchResultSection section = 1;
+}
+
+// Etmpty request to get version information.
```
Comment (beets) [inline proto/misc.proto:59]:
```suggestion
// Empty request to get version information.
```

**Review by beets — APPROVED** [review]
thanks for this!

were the protos split according to the api route? it would be nice to have a comment at the top of each file noting what should go in there.

**Comment by shifucun** [comment]
> thanks for this!
> 
> were the protos split according to the api route? it would be nice to have a comment at the top of each file noting what should go in there.

Very good idea, added the REST URL as file comment

File: proto/misc.proto (hunk 2 above)
Comment (shifucun) [reply proto/misc.proto]: Done

File: proto/landing.proto (hunk 1 above)
Comment (shifucun) [reply proto/landing.proto:1]: I am thinking putting all the landing page in this file, and have naming /landing/place(protein | disease). what do you think?

Comment (beets) [reply proto/landing.proto:1]: yea, that sounds good. we can further split it up if it becomes too big..

Comment (shifucun) [reply proto/landing.proto]: Yep, updated to /internal/place(protein) per discussion offline

**Comment by shifucun** [comment]
Thanks for review and suggestions!

**Pull request merged**